### PR TITLE
Add configuration for service account id/secret

### PR DESCRIPTION
**Pull request by sbose78**
- add sample openshift secrets
- add config vars

### CODE REVIEW

File: openshift/f8notification.secrets.yaml [hunk 1]
```diff
@@ -19,4 +19,6 @@ objects:
   data:
     wit.api: https://api.prod-preview.openshift.io
     keycloak.url: https://sso.prod-preview.openshift.io
-    keycloak.realm: fabric8-test
\ No newline at end of file
+    keycloak.realm: fabric8-test
+    service.account.id: 4c83ca2d-6dcc-41c9-ac7d-b068ad4d17c5
```
Comment (vpavlin) [inline openshift/f8notification.secrets.yaml]: service.account.* are secrets (at least the service.account.secret is:) ). Could you please move them to the kind: Secret section?

File: openshift/f8notification.secrets.yaml [hunk 2]
```diff
@@ -11,6 +11,8 @@ objects:
   type: Opaque
   data:
     madrill.api: eHh4eHh4
+    service.account.id: 4c83ca2d-6dcc-41c9-ac7d-b068ad4d17c5
```
Comment (vpavlin) [inline openshift/f8notification.secrets.yaml]: Secrets need to be `base64` encoded
i.e. 

```
echo -n "$VALUE" | base64 -w 0
```

**Comment by sbose78** [comment]
@alexeykazakov I'm OK if we merge this PR with the rest of my associated changes in this PR, Or if we merge this now and do the changes in another PR.

**Comment by alexeykazakov** [comment]
[test]

**Pull request merged**